src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/ReviewConfirmModal/TokensSectionDiff.tsx

```diff
@@ -58,11 +63,16 @@ export function TokensSectionDiff() {
 
   const formattedReservedRate = parseFloat(formatReservedRate(newReservedRate))
 
+  const currentMintRateAfterDiscountRateApplied = deriveNextIssuanceRate({
+    weight: BigNumber.from(0),
+    previousFC: currentFundingCycle,
+  })
+
   return (
     <DiffSection
       content={
         <div className="mb-5 flex flex-col gap-3 text-sm">
-          {mintRateHasDiff && currentMintRate && (
+          {mintRateHasDiff && currentMintRateAfterDiscountRateApplied && (
```

Comment:
This needed changing, otherwise in that Artizen case, you end up with "Old value: 101" and "New value: 101". This way, we set the old value to be diffed to the current mintRate after discountRate applied.